client/my-sites/pages/page/index.js

```diff
@@ -736,6 +738,15 @@ class Page extends Component {
 		this.props.recordEvent( 'Clicked Delete Page' );
 	};
 
+	recordPageTitleClick = ( canEdit ) => {
+		this.props.recordTracksEvent( 'calypso_pages_page_title_click', {
+			page_type: 'real',
+			blog_id: this.props.siteId,
+			can_edit: canEdit,
+		} );
+		this.props.recordPageTitle();
```

Comment:
I think we can create a "helper" method in this file instead, similar to the helpers.js

```
recordGoogleEvent = (action) => {
  this.props.recordGoogleEvent('Pages', action);
}
```

then we can reuse it in other methods, what do you think?

Reply:
Don't we have this method from helpers? Thanks

https://github.com/Automattic/wp-calypso/blob/3e86e0384bd16b334a56505a198b340efdc0dc44/client/my-sites/pages/page/index.js#L720

Reply:
Yes but now it's ambiguous as we have Tracks events as well, so I'd rename it to `recordGoogleEvent()` if we want to reuse it

Reply:
OKay, let's rename the helper method, I can see it is used on pages and virtual-pages components

Reply:
@fushar I will address the suggested approach in this follow-up PR #70626